scripts/ci/run-local-chain-nettest.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2020 Tocqueville Group
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+# This script runs given nettest scenario using local-chain with the node
+# on '127.0.0.1:8734' (currently used rpc address for local-chain jupiter node).
+# This script expects 'tezos-client' to be in PATH.
+set -euo pipefail
+
+nettest_scenario="$1"
+
+node_addr="$NETTEST_NODE_ADDR"
+node_port="$NETTEST_NODE_PORT"
+
+TEMPDIR="$(mktemp -d --tmpdir="$PWD")"
+tezos-client -A "$node_addr" -P "$node_port" -d "$TEMPDIR" import secret key nettest \
```

Comment:
That's fine for now, but later I think we should do https://gitlab.com/morley-framework/morley/-/issues/223 and simplify this script.